upstream-community-operators/kiali/kiali.v0.18.1.clusterserviceversion.yaml

```diff
@@ -206,7 +206,7 @@ spec:
                     fieldRef:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
-                  value: "kiali-operator"
+                  value: "Kiali Operator"
```

Comment:
@galletti94 The build is failing probably because of the change in the `OPERATOR_NAME` value. This env variable is used by the operator internally. Could you change it back to `kiali-operator` and see if the build passes?